pkg/controllers/node/termination/terminator/suite_test.go

```diff
@@ -60,6 +60,7 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	fakeClock = clock.NewFakeClock(time.Now())
```

Comment:
Why did you add the fakeClock here -- doesn't look like it is being used anywhere in this testing?

Reply:
It's passed down to the terminator and was causing nil panics. We could probably scope it down and just pass it there, but I don't mind keeping it test global so it's obvious to use the fake clock passed to the terminator rather than a new one in any future tests that use it.